ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -160,7 +160,7 @@ class JITModBuilder {
   // before the control point) to the extracted values (appearing after the
   // control point). This map helps to match inserted values to their
   // corresponding extracted values using their index in the struct.
-  std::map<uint64_t, Value *> InsertValueMap;
+  std::map<uint64_t, Value *> UpsideDownMap;
```

Comment:
Sure this is better than `CPIndexMap` or just `IndexMap`? `UpsideDownMap` sounds like something from a Netflix show.

Reply:
I must admit the whole "upside down" stuff doesn't make sense to me. I suspect I'm missing something obvious, but I'm not quite sure why. I wonder if "reverse order" or similar would be clearer?

Reply:
How about `LiveIndexMap`?

Reply:
I can live with that.

Reply:
@vext01 In terms of the general "upside down", would "reverse order" be correct? I still don't quite understand what "upside down" means here.

Reply:
It's to do with the disparity between the order of execution between AOT code and code collected by tracing.

Suppose the AOT code is:
```
bb1:
  <stuff1>
  control_point()
  <stuff2>
  goto bb1
```

since traces start after the control point, and end before it, a trace of one iteration of the above loop will be like:

```
<stuff2>
<stuff1>
```

So the code is "upside-down". There is an attempt to explain this in [this comment](https://github.com/ykjit/yk/pull/452/files#diff-b76ed8a29ed650e7c2a44a5809bc05a22cca958d0b9f5532dab74cce8503d112R669).

This map we are talking about here is needed (without going into mind-boggling detail) to deal with the case where variables are undefined in `<stuff2>` if you don't have the context of `<stuff1>` to tell you where they came from.

So.. propose a name :)

Reply:
Ah! I think it would be clearer to say that "a traced loop begins at the control point; if, as is likely, the control point is not at the beginning of the surrounding `while` loop, code between the start of that `while` loop and the control point will be recorded at the end of the loop." I think I'd drop the "upside down" thing entirely (even though I sort-of now see where you're coming from).

Reply:
But what is the name of the map to be?

Reply:
`LiveIndexMap` is fine. I also wonder if I can be cheeky and ask if you'd also consider rephrasing the comments to remove "upside down" so that thickies like me aren't confused in the future?

Reply:
Yep, that's fine.

Reply:
Thanks!

Reply:
@ptersilie I still think this logic could be simplified now that the struct is passed by pointer. One for another PR though.

Reply:
@ltratt try 2df8c8a

Reply:
Modulo the typo, that's fine!

Reply:
760691d fixes typo

Reply:
This looks good.